src/overlays/actors/ovl_En_Kakasi/z_en_kakasi.c

```diff
@@ -1101,7 +1101,7 @@ void EnKakasi_IdleRisen(EnKakasi* this, GlobalContext* globalCtx) {
 void EnKakasi_RisenDialogue(EnKakasi* this, GlobalContext* globalCtx) {
     Math_SmoothStepToS(&this->actor.shape.rot.y, this->actor.yawTowardsPlayer, 5, 1000, 0);
 
-    if (Message_GetState(&globalCtx->msgCtx) == 5 && Message_ShouldAdvance(globalCtx) != 0) {
+    if (Message_GetState(&globalCtx->msgCtx) == 5 && Message_ShouldAdvance(globalCtx)) {
```

Comment:
```suggestion
    if ((Message_GetState(&globalCtx->msgCtx) == 5) && Message_ShouldAdvance(globalCtx)) {
```
and probably more in this file.

Reply:
I'll change the brackets that are suggested here, but I don't want to go hunting these down until there's a consensus on style, since we haven't reached one yet (or have we?) If we have, then that's probably a separate PR cleaning these up

Reply:
It's in #793 officially now, haven't had anyone disagreeing with it there yet.